File: FreeCodeCamp/Javascript/reverseString.js
// https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/reverse-a-string
// Reprak11 
function reverseString(str) {
    let temp = []
    str.split("").forEach(element => {
       temp.unshift(element); 
    });
    str = temp.join("")
    return str;
}

function reverseString2(str){
    let expre = /./ig;
    str = str.match(expre);
    return str.reverse().join("");
}


console.log(reverseString2(prompt()));

File: FreeCodeCamp/Python/reverseString.py
# https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-algorithm-scripting/reverse-a-string
# Reprak11

def reverseString(str):
    temp = ""
    for i in reversed(range(len(str))):
        temp += str[i]
    str = temp
    return str;

if __name__ == "__main__":
    print(reverseString(input()))